.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+**Description**
+
+<!-- 
+Please include a summary of the changes and which issue is fixed or which feature it added.
+Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
+
+<!--
+If Connected to an issue, include:
+Closes # (issue)
+-->
+
+**Testing**
+
+<!-- Please shortly describe how you tested your code and mark all you have done after -->
+
+<!-- exclude any of the following if they do not apply -->
+- [ ] I added basic working examples (possibly in doc-comment)
+- [ ] I added tests for large (pointer representation) values
+- [ ] I triggered all possible errors in my test in every possible way
+- [ ] I included assertions in which I compare expected values with computed values
```

Comment:
It would be possible to write a test without checking the value.
For example, just test with unwrap that no error is created.

But I think that  how assertions have to be done is not necessary to specify. Just saying that the features has to be tested should be enough.
```suggestion
- [ ] I included tests for all reasonable edge cases
```